makefile

```diff
@@ -84,45 +89,44 @@ tests:
 	go test -v ./internal/...
 	go test -v ./pkg/...
 
-e2e-start-cluster:
-	./e2e/setup/setup_cluster_ci.sh
-	./e2e/setup/setup_kube_config_ci.sh
-	KUBECONFIG=.kube/config kubectl apply -f ./e2e/setup/namespace-with-anonymous-user.yaml
-	#docker-compose -f ./e2e/setup/kafka/docker-compose.yaml up -d
-	docker run -d --name nats -p 4223:4223 -p 8223:8223 nats-streaming -p 4223 -m 8223
-
-e2e-stop-cluster:
-	docker stop kube nats
-	docker rm kube nats
-	#docker-compose -f ./e2e/setup/kafka/docker-compose.yaml down
-
-.ONESHELL:
-tests-e2e: e2e-start-cluster build-docker
-	docker run -d --name redis -p=6379:6379 redis
-	docker run -d --name server --network=host -p=50051:50051 \
-		-v $(shell pwd)/e2e:/e2e \
-		armada ./server --config /e2e/setup/insecure-armada-auth-config.yaml --config /e2e/setup/nats/armada-config.yaml
-	docker run -d --name executor --network=host -v $(shell pwd)/.kube/config:/kube/config \
-		-e KUBECONFIG=/kube/config \
-		-e ARMADA_KUBERNETES_IMPERSONATEUSERS=true \
-		-e ARMADA_KUBERNETES_STUCKPODEXPIRY=15s \
-		armada-executor
-	function tearDown {
-		echo -e "\nexecutor logs:"
-		docker logs executor
-		echo -e "\nserver logs:"
-		docker logs server
-		docker stop executor server redis
-		docker rm executor server redis
-	}
-	trap tearDown EXIT
-	sleep 10
-	echo -e "\nrunning test:"
+e2e-create-cluster: build-docker-e2e-tests
+	go get sigs.k8s.io/kind@v0.11.1
+	kind create cluster --config e2e/setup/kind-confg-server.yaml
+	kubectl apply -f ./e2e/setup/namespace-with-anonymous-user.yaml
+	helm repo add nats https://nats-io.github.io/k8s/helm/charts
+	helm repo add bitnami https://charts.bitnami.com/bitnami
+	helm repo update
+	helm install redis bitnami/redis \
+		--set architecture=standalone \
+		--set auth.sentinel=false \
+		--set auth.enabled=false \
+		--wait
+	helm install nats nats/stan --wait
+	kind load docker-image tests/armada-server:latest --name e2e
+	kind load docker-image tests/armada-executor:latest --name e2e	
+
+e2e-install-armada:
+	find . \( -name "Chart.yaml" -o -name "values.yaml" \) -exec sed -i s/LATEST/1.0.0/ {} +
```

Comment:
note: see if we can use here `yq` to replace this values and get some return exit code if it's successful or not